File: ExcelDivider/src/main/java/ru/tsystems/divider/FileClear.java
package ru.tsystems.divider;

import java.io.FileNotFoundException;
import java.io.IOException;

import ru.tsystems.divider.utils.impl.FileCleaner;
import ru.tsystems.divider.exceptions.NoShetException;

public class FileClear {

    public static void main(String[] args) {
        try {
            FileCleaner fileCleaner = new FileCleaner();
            fileCleaner.clearFile(args[0], args[1], args[2]);
        } catch (FileNotFoundException fnfExc) {
            System.out.println("No file: " + fnfExc.getMessage());
        } catch (IOException ioExc) {
            System.out.println("File problem" + ioExc.getMessage());
        } catch (NoShetException nsExc) {
            System.out.println("Sheet exception: " + nsExc.getMessage());
        }
    }
}

File: ExcelDivider/src/main/kt/ru/tsystems/divider/FileClear.kt
package ru.tsystems.divider

import ru.tsystems.divider.exceptions.NoShetException
import ru.tsystems.divider.utils.impl.FileCleaner
import java.io.FileNotFoundException
import java.io.IOException

class FileClear {

    fun main(args: Array<String>) {
        try {
            val fileCleaner = FileCleaner()
            fileCleaner.clearFile(args[0], args[1], args[2])
        } catch (fnfExc: FileNotFoundException) {
            println("No file: " + fnfExc.message)
        } catch (ioExc: IOException) {
            println("File problem" + ioExc.message)
        } catch (nsExc: NoShetException) {
            println("Sheet exception: " + nsExc.message)
        }

    }
}